.github/workflows/build_and_test.yml

```diff
@@ -260,6 +244,68 @@ jobs:
         name: unit-tests-log-${{ matrix.modules }}--1.8-hadoop3.2-hive2.3
         path: "**/target/unit-tests.log"
 
+  sparkr:
+    name: Build modules - sparkr
+    runs-on: ubuntu-20.04
+    container:
+      image: dongjoon/apache-spark-github-action-image:20201025
```

Comment:
nit: how about adding some comments here about what the image pre-installs? e.g., which version of R is installed (and more?)

Reply:
I agree that is a good idea. Like we did for `dev/create-release/spark-rm/Dockerfile`, can we put the description into the original `Dockerfile`? For now, it's not documented explicitly here and there (Dockerfile).
- https://github.com/dongjoon-hyun/ApacheSparkGitHubActionImage/blob/main/Dockerfile

I hope we can convert `ApacheSparkGitHubActionImage` into Apache Spark branch.

Reply:
> Like we did for dev/create-release/spark-rm/Dockerfile, can we put the description into the original Dockerfile? For now, it's not documented explicitly here and there (Dockerfile).

Ah, ok. I see. That looks okay.